environment.yml

```diff
@@ -37,6 +37,9 @@ dependencies:
   - pygments
   - pydot
   - ipython
+  - pip
```

Comment:
It can be installed from conda forge too, no need to include pip and install from github

Reply:
The one in the doc folder should also be updated then. Link: https://anaconda.org/conda-forge/pymc-sphinx-theme